src/services/brandService.ts

```diff
@@ -69,49 +69,117 @@ export const createBrandService = async (name: string, file: Express.Multer.File
     }
 };
 
-// function to download an image from Firebase Storage
-const downloadImageFromFirebase = async (fileName: string): Promise<Buffer> => {
-    // Create a reference to the file in Firebase Storage using the provided file path
+const getBrandImageUrlFromFirebase = async (imageUrl: string): Promise<string> => {
+    const fileName = imageUrl.split(`${bucket.name}/`)[1];
     const file = bucket.file(fileName);
+
+    try {
```

Comment:
Cant we have this code of fetching an image in some util?
@batooldshilleh what do you think?